monitoring/prometheus/prometheus.yml

```diff
@@ -6,3 +6,9 @@ scrape_configs:
   - job_name: 'prometheus' 
     static_configs: 
       - targets: ['localhost:9090']
+
+  - job_name: 'django-prometheus'
+    metrics_path: '/prometheus/metrics'
+    static_configs:
+      - targets: ['django:8000']
```

Comment:
Fetch this from env file. We will use evalai endpoint as target not django container port. This won't work in case we set up prometheus on a separate container.